### DISCUSSION TITLE
http://mapsforge.org/renderTheme page not found

### DISCUSSION THREAD

**Discussion by red0bear** [mapsforge/mapsforge#1389] [Q&A] [answered]
while try use those rendertheme with osmdroid it says i cant read from there those https://raw.githubusercontent.com/mapsforge/mapsforge/dev/resources/renderTheme-v4.xsd from there or where raw were. Can you help me please ?

**Comment by red0bear** [comment]
There is a way to set it locally ? seems raw is that one  https://github.com/mapsforge/mapsforge/blob/master/resources/renderTheme.xsd

**Comment by devemux86** [comment, marked as answer]
For Mapsforge see the [GettingStarted](https://github.com/mapsforge/mapsforge/blob/master/mapsforge-samples-android/src/main/java/org/mapsforge/samples/android/GettingStarted.java) example and the [mapsforge-samples-android](https://github.com/mapsforge/mapsforge/tree/master/mapsforge-samples-android).

For Osmdroid better ask in their forum.

Regarding the xml schema, it is used when writing themes to validate the syntax.
It can be downloaded locally and declare the local path in the theme.

**Comment by red0bear** [comment]
i made some examples https://github.com/red0bear/androidmapsexample to see if people see the problem.
